controlers/controlers.js

```diff
@@ -439,7 +454,60 @@ const uploadAvatar = async (req, res, next) => {
     console.error(err);
     next(err);
   }
- }
+}
+ 
+const verification = async (req, res, next) => {
+
+  const verificationToken = req.params.verificationToken;
+    try {
+      const user = await Users.findOne({ verificationToken: verificationToken })
+        if (!user) {
+            res.status(400)
+                .json(`User was not found`)
+        }
+        else if (user.verify === true) {
+            res.status(400)
+                .json(`User was verified`)
+        }
+        else {
+          user.setVerify(true);
+          user.setVerificationToken(null)
+          await user.save();
+          res.json({ message: 'Verification successful'});
+        }
+  } catch (err) {
+    console.error(err);
+    next(err);
+  }
+}
+
+const newVerificationEmail = async (req, res, next) => {
+  const { email} = req.body
+  console.log(email)
+
+  try {
+    const user = await Users.findOne({ email: email })
+    if (!user) {
+      res.status(400)
+        .json(`User was not found`)
+    }
+    else if (user.verify === true) {
```

Comment:
```suggestion
    else if (user.verify) {
```
To też zadziała, krótsza wersja